renovate.json

```
@@ -57,7 +57,8 @@
         ".github/actions/setup/action.yml"
       ],
       "matchStrings": [
-        "google-java-format\/releases\/download\/v(?<currentValue>.*?)\/google-java-format-(?<currentValue>.*?)-all-deps"
+        "google-java-format/releases/download/v(?<currentValue>.*?)/google-java-format-(.*?)-all-deps",
```

```suggestion
        "google-java-format/releases/download/v(?<currentValue>.*?)/google-java-format-",
```

We can avoid checking here